api-samples/userScripts/manifest.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "User Scripts API Demo",
```

Comment:
@oliverdunk Sorry forgot to ask, can we add a minimum_chrome_version?

Reply:
Done!